Core/RandomTest.cc

```diff
@@ -48,6 +54,45 @@ TEST(CoreRandomTest, fork) {
     EXPECT_GT(2U, failures) << failures;
 }
 
+void sleepWithLock(bool& haveLock, std::mutex& m, Core::ConditionVariable& c) {
+    LogCabin::Core::Random::acquireMutex();
+    {
+        std::unique_lock<std::mutex> lock(m);
+        haveLock = true;
+        c.notify_one();
+    }
+    usleep(10000);
+    LogCabin::Core::Random::releaseMutex();
+}
+
+TEST(CoreRandomTest, forkWillNotDeadlock) {
```

Comment:
Rename to forkWillNotDeadlock_TimingSensitive since the test could miss failures on an overloaded system.